CRM/Report/Form/Contribute/PCP.php

```diff
@@ -190,6 +211,16 @@ public function from() {
 LEFT JOIN civicrm_contribution_page {$this->_aliases['civicrm_contribution_page']}
           ON {$this->_aliases['civicrm_pcp']}.page_id =
              {$this->_aliases['civicrm_contribution_page']}.id";
+
+    // for credit card type
+    if ($this->isTableSelected('civicrm_financial_trxn')) {
```

Comment:
Yes, like I have suggested in https://github.com/civicrm/civicrm-core/pull/9988 but I think we should address it in a separate issue and as other tables needs the same fix.